### PR TITLE
Add copyPixelsToBuffer/copyPixelsFromBuffer to ShadowBitmap.

### PR DESCRIPTION
**Pull request by timonvo**
The support is currently limited to Buffers of type ByteBuffers and 4-byte-per-pixel Bitmaps.

### CODE REVIEW

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java [hunk 1]
```diff
@@ -521,6 +523,52 @@ public static Bitmap nativeCreateFromParcel(Parcel p) {
     return createBitmap(parceledColors, parceledWidth, parceledHeight, parceledConfig);
   }
 
+  @Implementation
+  public void copyPixelsFromBuffer (Buffer dst) {
```
Comment (jongerrish) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java]: Can you remove whitespace before the parenthesis. Same below.

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java [hunk 2]
```diff
@@ -521,6 +523,52 @@ public static Bitmap nativeCreateFromParcel(Parcel p) {
...
+    if (isRecycled()) {
+      throw new IllegalStateException("Can't call copyPixelsFromBuffer() on a recycled bitmap");
+    }
+
+    // See the related comment in #copyPixelsToBuffer(Buffer).
+    if (getBytesPerPixel(config) != 4) {
```
Comment (jongerrish) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java]: Make 4 a constant?

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java (hunk 1 above)
Comment (timonvo) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java]: Done.

File: robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java (hunk 2 above)
Comment (timonvo) [inline robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBitmap.java]: Done.

**Comment by timonvo** [comment]
Friendly ping! @jongerrish or someone else, can you review this change?

**Comment by jongerrish** [comment]
Looks like the build just kicked off again. I couldn't merge this last week because of a build failure. I'll merge if it goes green.

**Comment by timonvo** [comment]
Thanks!

**Pull request merged**